src/toolbars/toolbar-component.js

```diff
@@ -52,6 +52,7 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
 
     function viewSelected (viewId) {
       ctrl.config.viewsConfig.currentView = viewId;
+      ctrl.config.isTableView = viewId === 'tableView';
```

Comment:
I would rather use the isTableView property on config but if a viewConfig is given, update that property based on the current view Id.